### DISCUSSION TITLE
9V Fast Charging Option for Huawei devices

### DISCUSSION THREAD

**Discussion by nthebach** [phhusson/treble_experimentations#1843] [Q&A]
Hi everyone, i have a huawei m5 pro tablet which supports 9V - 2A fast charging. The gsi has 2 extra charging options for huawei devices which are mid-fast (5V - 2.75A) and fast (5V - 4A). Is there any way to add 9V fast charging support for the device?

**Comment by phhusson** [comment]
Have you actually checked whether it goes to 9V or not? I'd say it does

15 mai 2021 14:56:57 techyescountry ***@***.***>:

**Reply by nthebach** [reply]
Thank you for the prompt reply. I cannot tell for sure to be honest because I dont have proper equipment to measure the voltage. But based on my observation, it take 30 mins to charge 15% on gsi with Mid-fast (2750mA) enabled while it takes only 15 mins on  stock emui 9 rom.
